tests/ci/release.py

```diff
@@ -150,35 +240,38 @@ def _bump_prestable_version(self, release_branch: str, args: argparse.Namespace)
         self.run(
             f"git commit -m 'Update version to {new_version.string}' '{cmake_path}'"
         )
-        with self._push(release_branch, args):
+        with self._push(self.release_branch):
             with self._create_gh_label(
-                f"v{release_branch}-must-backport", "10dbed", args
+                f"v{self.release_branch}-must-backport", "10dbed"
```

Comment:
add enum for colors

Reply:
Renamed the argument to `color_hex` and added a comment to `_create_gh_label`